Description: ntpdig: Only show socket error messages in debug
 This is a continuation on the changes from
 commit 42ec668ddfb82606d5aea379cc74845084b10d15.
 .
 As in #680, these error messages are confusing to users who do not
 have IPv6.  This came up again in:
 https://bugs.debian.org/1020865
 .
 If all queries fail, the user still gets "ntpdig: no eligible servers",
 so they aren't left wondering whether it worked or not.
Forwarded: https://gitlab.com/NTPsec/ntpsec/-/merge_requests/1288.patch
Author: Richard Laager <rlaager@wiktel.com>
Bug-Debian: https://bugs.debian.org/971523
Last-Update: 2022-09-27

--- a/ntpclients/ntpdig.py
+++ b/ntpclients/ntpdig.py
@@ -115,7 +115,8 @@
         try:
             s.sendto(packet, sockaddr)
         except socket.error as e:
-            log("socket error on transmission: %s" % e)
+            if debug:
+                log("socket error on transmission: %s" % e)
             continue
         if debug >= 2:
             log("Sent to %s:" % (sockaddr[0],))
